cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocConf.java

```diff
@@ -213,7 +213,7 @@ private DataprocConf(@Nullable String accountKey, String region, String zone, St
     this.pollDeleteDelay = pollDeleteDelay;
     this.pollInterval = pollInterval;
     this.encryptionKeyName = encryptionKeyName;
-    this.gcsBucket = gcsBucket;
+    this.gcsBucket = DataprocUtils.getBucketName(gcsBucket);
```

Comment:
Instead of adding here, 
Add this logic in line 638. 
`String gcpCmekBucket = getString(properties, "gcsBucket");`  in the create function.

Reply:
Done !!